### ISSUE TITLE
The screen reader does not announce the open and close state of the keyboard instruction button.

### ISSUE THREAD

**Issue by Uditsingh67** [cengage/react-magma#427]
**To Reproduce**

1. Launch https://storybook-preview-290--upbeat-sinoussi-f675aa.netlify.app/?path=/story/chart--default
2. Click on the chart
3. Select the default chart
4. Navigate to the keyboard instructions button and verify

**Expected behavior**
The screen reader must announce the open and close state of the keyboard instruction button.

**Screenshots**
 https://www.screenpresso.com/=wSN2g

**Desktop (please complete the following information):**
Windows 10 : Chrome [ 95.0.4638.69] /Edge [ Version 95.0.1020.40]

**Additional context**
Issue is reproducible on all the charts.
**WCAG: 1.3.1 Info and Relationship**
https://www.w3.org/TR/UNDERSTANDING-WCAG20/content-structure-separation-programmatic.html

**Comment by rcgillet** [comment]
Updated

https://storybook-preview-290--upbeat-sinoussi-f675aa.netlify.app

**Comment by hkrishna29** [comment]
The issue is fixed. Screen-cast : https://www.screenpresso.com/=BZ3Vf

**Comment by hkrishna29** [comment]
the issue has been fixed
